scripting/pvpoptin.sp

```diff
@@ -24,7 +24,7 @@
 //#tryinclude <piggyback>
 #define REQUIRE_PLUGIN
 
-#define PLUGIN_VERSION "24w17a"
+#define PLUGIN_VERSION "23w12a"

```

Comment:
What happened here btw? Why did the version regress from develop to main?

Reply:
Result of porting over the changes from master to develop, ignore.

Reply:
That would explain why your gamedata have been out of date. I really need to merge dev back into main... 😅

Reply:
i replaced the gamedata entirely though with the dev one